metaflow/plugins/aws/batch/batch.py

```diff
@@ -262,6 +263,11 @@ def create_job(
         if tmpfs_enabled and tmpfs_tempdir:
             job.environment_variable("METAFLOW_TEMPDIR", tmpfs_path)
 
+        if S3_UPLOAD_ARGS is not None:
```

Comment:
It looks like this will always be an empty dictionary {} so maybe just take out this conditional

Reply:
I'm wondering now whether we even need a default value for the s3 upload args? The original reason for the {} default was to enable parsing of an environment variable passed as a JSON string. Not sure it's necessary now that we pass the upload args individually.

Reply:
Cool, it looks fine to me to change it back so no config will be None